lib/src/Uncertainty/Algorithm/Simulation/openturns/NAIS.hxx

```diff
@@ -82,7 +103,18 @@ private:
 
   // Result of NAIS algorithm
   NAISResult naisResult_;
-
+  
+  // some results
+  UnsignedInteger numberOfSteps_ = 0;// number of subset steps
+  Point thresholdPerStep_;// intermediate thresholds
+
+
+  // keep samples generated at each step
+  Bool keepSample_ = false;
+  PersistentCollection<Sample> inputSample_;
+  PersistentCollection<Sample> outputSample_;
```

Comment:
these must referenced in the save/load methods

Reply:
not really sure to understand your remark. I copied the lines from subset sampling and pasted them at the same location.

Reply:
You mean that you want the creation of save and load methods in NAIS to use the storatemanager. Currently, there is no such methods in the class.

Reply:
ok, nevermind